Detectors/TPC/simulation/include/TPCSimulation/Detector.h

```diff
@@ -133,7 +133,7 @@ class Detector: public o2::Base::DetImpl<Detector> {
     void ConstructTPCGeometry();
 
     /** Define the sensitive volumes of the geometry */
-    void DefineSensitiveVolumes();
+    void DefineSensitiveVolumes() override;
```

Comment:
This seems to be a new virtual interface in FairRoot (which did not exist until Mar 26).
The change is probably ok but the function might be now be called directly from FairRoot and we have to make sure that this is what we want.

Reply:
Indeed, it is called from FairDetector::Initialize and only in MT mode. @ihrivnac, this is your commit, could you please clarify why it is done in the MT mode only and what does this change now for the O2 detectors? Currently every O2 detector calls internally (from Initialize) its own defineSensitiveVolumes() (note the lower case for all detectors except the TPC which for this reason clashed with FairDetector::DefineSensitiveVolumes()) and in the end calls FairDetector::Initialize.